### DISCUSSION TITLE
[Camera Support]: ffmjpeg vs go2rtc what should I choose?

### DISCUSSION THREAD

**Discussion by nguyentrangia** [blakeblackshear/frigate#10860] [Camera Support]
### Describe the problem you are having

I can config camera using ffmjpeg or go2rtc streams. Which one is better in term of latency and performance? Sorry for the very noob question.

### Version

0.13.2

### Frigate config file

```yaml
I guess this question doesn't need this part
```

### Relevant log output

```shell
I guess this question doesn't need this part
```

### FFprobe output from your camera

```shell
Stream 0:
Return Code: 0

Video:

Codec: H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10
Resolution: 640x480
FPS: 25/1

Audio:

Codec: AAC (Advanced Audio Coding)

Audio:

Codec:
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Imou

**Comment by NickM-27** [comment]
You mean ffmpeg? You have to use ffmpeg. Using go2rtc is needed for some features like live view, restreaming, etc.
